lib/index.d.ts

```diff
@@ -561,6 +561,11 @@ declare class Realm {
      */
     static openAsync(config: Realm.Configuration, callback: (error: any, realm: Realm) => void, progressCallback?: Realm.Sync.ProgressNotificationCallback): void
 
+    /**
+     * Open the default Realm.
+     */
+    static default(): ProgressPromise;
```

Comment:
Shouldn’t this be `ProgressPromise<Realm>`?

Reply:
`interface ProgressPromise extends Promise<Realm>`